mkdocs.yml

```diff
@@ -69,7 +69,9 @@ nav:
   - Calling base methods: calling-base-methods.md
   - Calling wrapped methods: calling-wrapped-methods.md
   - Changing behavior between calls: changing-behavior-between-calls.md
-- Argument constraints: argument-constraints.md
+- Argument constraints:
+  - Argument constraints: argument-constraints.md
+  - Capturing arguments: capturing-arguments.md

```

Comment:
I'm not sure I would put this under "argument constraint", because as I mentioned before, it's not necessarily related.